### ISSUE TITLE
NIP-94 support?

### ISSUE THREAD

**Issue by cxplay** [hzrd149/blossom#18]
When a media file is attached to an event, for better retrieval and filtering it should be considered to add file metadata, partly provided by the Blossom server and partly by the client (especially Alt text).
